pkg/proxy/winkernel/proxier.go

```diff
@@ -106,8 +106,8 @@ type loadBalancerFlags struct {
 	isIPv6          bool
 }
 
-// internal struct for string service information
-type serviceInfo struct {
+// ServiceInfo is an internal struct for string service information
+type ServiceInfo struct {
```

Comment:
What's the reason for exporting this struct? It seems specific to winkernel so should remain private

Reply:
ah yes, see comment below just dug it up